server/views/layout/default.njk

```diff
@@ -51,6 +51,71 @@
       {% block body %}
       {% endblock %}
     </div>
+
+    {% set footerSocial = {
+      columns: [
+        {
+          items: [
+            {
+              url: "https://twitter.com/explorewellcome",
+              title: "Twitter",
+              service: "Twitter",
+              icon: "social/twitter"
+            },
+            {
+              url: "https://www.facebook.com/wellcomecollection/",
+              title: "Facebook",
+              service: "Facebook",
+              icon: "social/facebook"
+            }
+          ]
+        },
+        {
+          items: [
+            {
+              url: "https://www.instagram.com/wellcomecollection/",
+              title: "Instagram",
+              service: "Instagram",
+              icon: "social/instagram"
+            },
+            {
+              url: "https://soundcloud.com/wellcomecollection",
+              title: "Soundcloud",
+              service: "Soundcloud",
+              icon: "social/soundcloud"
+            }
+          ]
+        },
+        {
+          items: [
+            {
+              url: "http://wellcomecollection.tumblr.com/",
+              title: "Tumblr",
+              service: "Tumblr",
+              icon: "social/tumblr"
+            },
+            {
+              url: "https://www.youtube.com/user/WellcomeCollection",
+              title: "YouTube",
+              service: "YouTube",
+              icon: "social/youtube"
+            }
+          ]
+        },
+        {
+          items: [
+            {
+              url: "https://www.tripadvisor.co.uk/Attraction_Review-g186338-d662065-Reviews-Wellcome_Collection-London_England.html",
+              title: "TripAdvisor",
+              service: "TripAdvisor",
+              icon: "social/tripadvisor"
+            }
+          ]
+        }
+      }
+    ] %}
```

Comment:
Just wondering what the thinking was of having the footer to be able to take these as an argument, which seems to imply we might have different footers?

Could this not just be set in the footer?